src/features/diary/login/ui/Login.tsx

```diff
@@ -0,0 +1,52 @@
+import styles from './Login.module.scss';
+import { Button, Checkbox, Form, Input } from 'antd';
+import { routes } from 'src/shared/routes';
+
+export const Login = () => {
+  const onFinish = (values: any) => {
```

Comment:
do not use any, all types must be described, and if any is needed, then add a comment why any is needed